gflownet.py

```diff
@@ -396,9 +397,12 @@ def get_training_data(self, batch_size):
         rewards = self.env.get_reward(input_reward, done)
         terminal_states = rewards[rewards != 0]
         # mean of just the terminal states
-        self.logger.log_metric("rewards", np.mean(terminal_states))
+        self.logger.log_metric("mean_reward", np.mean(terminal_states))
+        self.logger.log_metric("max_reward", np.max(terminal_states))
         proxy_vals = self.env.reward2acq(terminal_states)
-        self.logger.log_metric("proxy_vals", np.mean(proxy_vals))
+        self.logger.log_metric("mean_proxy_score", np.mean(proxy_vals))
+        self.logger.log_metric("min_proxy_score", np.min(proxy_vals))
```

Comment:
It would also be possible to first compute : 
1. The acquisition function values : using env.get_acq 
2. The reward function values : env.acq2reward
We would store the logs in the "natural" order of computations.

Reply:
In the current implementation env.get_reward() returns the final reward which is to be fed to the policy network. It applies the acq as well as the power/boltzmann function. Do you suggest adding two different functions? One that just computes the acquisition value, and the other that applies the boltzmann? I feel that might be a little confusing. Like right now we can simply call env.get_reward() and get the final reward that must be fed to the policy.
Hmm, not sure it makes a difference if we don't follow the "natural" order of computation while logging.